src/pfe/portal/package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "start": "./npm-start.sh",
     "prod": "node server.js",
-    "dev": "nodemon server.js",
+    "dev": "node --inspect=0.0.0.0:9777 server.js",
```

Comment:
Can we keep nodemon?

Reply:
I've changed it to keep nodemon and use your `devfile.env`.